### DISCUSSION TITLE
Increasing the size of the reference counter box

### DISCUSSION THREAD

**Discussion by TfTHacker** [TfTHacker/obsidian42-strange-new-worlds#18] [Show and tell]
All aspects of SNW can be modified with custom CSS snippets. The following will on mobile devices increase the font size for the display of the number and make the box wider:

```
.is-mobile .snw-reference {
    font-size: 10pt;
    padding-left: 10px;
    padding-right: 10px;
}
```
Of course you can increase or decrease the font size or reduce the margins.

`.snw-reference` is the root CSS class for the box. Adding `.is-mobile` tells Obsidian this change should only be applied to the mobile version of Obsidian. If you remove `.is-mobile`, both desktop and mobile will have these changes. For example:

```
snw-reference {
    font-size: 10pt;
    padding-left: 10px;
    padding-right: 10px;
}
```

**Comment by StirlingBaker** [comment]
For whatever reason, the CSS above didn't work for me @TfTHacker 

I ended up going with this, which works as I'd hoped in both the sidepanel and the popover. 

```
/* STRANGE NEW WORLDS */

.snw-ref-title-popover-label {
  font-size: .75em;
}

.snw-ref-item-container {
  font-size: .75em;
  padding-left: 10px;
  padding-right: 10px;
}
```
